templates/guides/vault-secrets-data-sources.md.tmpl

```diff
@@ -0,0 +1,17 @@
+---
+subcategory: ""
```

Comment:
are these supposed to have a subcategory now? or just not the guides?

cc @aidan-mundy

Reply:
I think the guides need to be at the top level, otherwise it will duplicate the folders w/ resource and data source docs in them.